### ISSUE TITLE
Focused element not visible

### ISSUE THREAD

**Issue by EmmanuelDemey** [EmmanuelDemey/GlobalIssueTracker#19]
# Informations générales

* URL du site : https://www.devoxx.co.uk/

# Descriptions

When an element has the focus, its look n feel should not be the same . In the next screenshot, the current focused element if the Track link on the main menu. The only way to have this information is to have a look to the URL on the bottom left. 

![image](https://user-images.githubusercontent.com/555768/154524583-ea27491c-4ed7-4e71-8d88-c63dacebe5ad.png)

On the main menu, you can maybe add the same orange when an text is focused

![image](https://user-images.githubusercontent.com/555768/154524840-62539578-22d6-4469-b2b6-bf501959a717.png)
